client/my-sites/stats/post-trends/style.scss

```diff
@@ -224,3 +189,13 @@ $mobile-container-margin: 16px;
 		}
 	}
 }
+
+.stats-heat-map__legend {
```

Comment:
Should this be defined here? Because this way it's being applied also to other sections, like the `All-time insights`.